### PR TITLE
FIX : used of internal makes lib unusable with netcore 3.0 +

### PR DESCRIPTION
**Pull request by Poltuu**
This pr fixes an issue the prevnts the lib from being used in 3.0.0 projects.

I didnot test it directly, as I faced some issues with trying to test it in 3.0 while supporting 2.1, hence previous PR.

### CODE REVIEW

**Comment by rducom** [comment]
Can you share the details about the mentioned issue in netcore 3 ?

**Comment by Poltuu** [comment]
This lib uses `Microsoft.AspNetCore.Mvc.Formatters.Internal.MediaTypeSegmentWithQuality`, which no longer exists in netcore 3.0 (namespace internal).
So this library cannot event be loaded and used. You get a TypeLoadedException.
The other approach to avoid this in the future is to target .NetStandard 2.0

**Comment by Poltuu** [comment]
https://github.com/aspnet/Announcements/issues/377

File: Lucca.Logs.AspnetCore/Lucca.Logs.AspnetCore.csproj [hunk 1]
```diff
@@ -1,16 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
```
Comment (Poltuu) [inline Lucca.Logs.AspnetCore/Lucca.Logs.AspnetCore.csproj]: why not simply netstandard2.0 ?

Comment (rducom) [reply Lucca.Logs.AspnetCore/Lucca.Logs.AspnetCore.csproj]: Because breaking changes in dependencies on aspnet core between 2.x & 3.x
image

**Pull request merged**